Vungle/VungleInterstitialCustomEvent.m

```diff
@@ -111,7 +109,7 @@ - (void)presentAdFromViewController:(UIViewController *)viewController
     }
 }
 
-- (void)dealloc
+- (void)cleanUp
```

Comment:
In MoPub 5.13.0 SDK, they changed the way they managed adapters and let the iOS system clean up the adapters automatically, so we need to manually remove reference for adapters by ourselves. Thanks a lot. 🙂